baked_in.go

```diff
@@ -2436,3 +2437,40 @@ func isDnsRFC1035LabelFormat(fl FieldLevel) bool {
 	val := fl.Field().String()
 	return dnsRegexRFC1035Label.MatchString(val)
 }
+
+func isCreditCard(fl FieldLevel) bool {
+	val := fl.Field().String()
+	var creditCard bytes.Buffer
+	parts := strings.Split(val, " ")
+	for _, part := range parts {
+		if len(part) < 2 {
+			return false
+		}
+		creditCard.WriteString(part)
+	}
+
+	ccDigits := strings.Split(creditCard.String(), "")
+	size := len(ccDigits)
+	if size < 14 || size > 16 {
```

Comment:
Globally, card numbers (PAN) can be 12 - 19 digits
(see https://en.wikipedia.org/wiki/Payment_card_number)

Reply:
@marrow16 thank you for pointing out the min and max size of a PAN. I've updated the code according to the documentation that you provided.

Reply:
@marrow16 @alessmar 
Hi folks,
Actually PAN length is in range 10-19

`IIN length has been extended to 8-digits in fifth edition of ISO/IEC 7812 published in 2017[[2]](https://en.wikipedia.org/wiki/Payment_card_number#cite_note-2) and PAN will continue to remain variable length, ranging from 10 to 19 digits.`